src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/google/HttpGoogleDataprocDAO.scala

```diff
@@ -187,7 +188,14 @@ class HttpGoogleDataprocDAO(appName: String,
     val request = oauth2.tokeninfo().setAccessToken(accessToken)
     retryWhen500orGoogleError(() => executeGoogleRequest(request)).map { tokenInfo =>
       (UserInfo(OAuth2BearerToken(accessToken), WorkbenchUserId(tokenInfo.getUserId), WorkbenchEmail(tokenInfo.getEmail), tokenInfo.getExpiresIn.toInt), Instant.now().plusSeconds(tokenInfo.getExpiresIn.toInt))
-    }.handleGoogleException(GoogleProject(""), Some("oauth"))
+    } recover {
+    case e: GoogleJsonResponseException =>
+      val msg = s"Call to Google OAuth API failed. Status: ${e.getStatusCode}. Message: ${e.getDetails.getMessage}"
+      logger.error(msg, e)
+      throw new WorkbenchException(msg, e)
+    case e: IllegalArgumentException =>
+      throw AuthorizationError()
```

Comment:
I'd like to have a comment here so we understand the context. Something like:

// Google throws IllegalArgumentException when passed an invalid token. Handle this case and rethrow a 401.

Also include a link to the github ticket.